src/lib/CasperClient.test.ts

```diff
@@ -0,0 +1,74 @@
+import { expect } from 'chai';
+
+import { SignatureAlgorithm } from './Keys';
+import { decodeBase16 } from './Conversions';
+import { Deploy } from './DeployUtil';
+import { CasperClient } from './CasperClient';
+
+let casperClient: CasperClient;
+describe('CasperClient', () => {
+  before(() => {
+    casperClient = new CasperClient('https://rpc.testnet.casperlabs.io/rpc');
```

Comment:
Is this URL in config somewhere? If so, could we please use that over hardcoding?